### PR TITLE
Fix github1734

### PR DESCRIPTION
**Pull request by greglandrum**
The problem was with the handling of multiple attachments at a chiral center. Once I tracked down the cause, the fix wasn't complicated.

### CODE REVIEW

File: Code/GraphMol/ChemTransforms/MolFragmenter.cpp [hunk 1]
```diff
@@ -357,26 +356,46 @@ namespace {
...
-    const Bond* obond = mol[*beg];
-    ++beg;
-    if (obond == bond) {
-      continue;
+  INT_LIST incomingOrder;
+  // since we may call this function more than once, we need to keep track of
+  // whether or not we've already been called and what the new atom order is.
+  // we do this with a property.
+  // this was github #1734
+  if (nAt->getPropIfPresent("_newBondOrder", incomingOrder)) {
```
Comment (bp-kelley) [inline Code/GraphMol/ChemTransforms/MolFragmenter.cpp]: We should add "_newBondOrder" to RDGeneral/types.h

Comment (bp-kelley) [reply Code/GraphMol/ChemTransforms/MolFragmenter.cpp]: Or make it a constant.  Should we remove these props at the end?  What happens the next time this is called?

Comment (greglandrum) [reply Code/GraphMol/ChemTransforms/MolFragmenter.cpp]: They are removed at the end when clearComputedProps() is called

Comment (greglandrum) [reply Code/GraphMol/ChemTransforms/MolFragmenter.cpp]: I will add it as a constant here (it’s not used elsewhere)

**Comment by greglandrum** [comment]
ok, those are done

**Pull request merged**